File: Seminar5/HomeWork/Model/Complex/Complex.java
package HomeWork.Model.Complex;

public class Complex {

    /**
     * real - реальная часть
     * image - мнимая часть
     */

    double real;
    double image;

    public Complex(double real, double image) {
        this.real = real;
        this.image = image;
    }

    public double getReal() {
        return real;
    }

    public void setReal(double real) {
        this.real = real;
    }

    public double getImage() {
        return image;
    }

    public void setImage(double image) {
        this.image = image;
    }

}

File: FinalTaskCalculator/Model/Complex/Complex.cs
namespace ModelComplex
{
    public class Complex
    {

        /**
         * real - реальная часть
         * image - мнимая часть
         */

        public double real;
        public double image;

        public double Real
        {
            get
            {
                return this.real;
            }
            set
            {
                this.real = value;
            }
        }
        public double Image
        {
            get
            {
                return this.image;
            }
            set
            {
                this.image = value;
            }
        }

        public Complex(double real, double image)
        {
            this.real = real;
            this.image = image;
        }
    }
}